BP/history/_BP_notes.info

```diff
@@ -0,0 +1,59 @@
+Move Bodh Gaya holy site to Bodh Gaya proper and move Mahabodhi Temple to it as well.
+
+Do Hispania after merging in Abian36's changes.
+
+Turkic/Arab succession laws needed for arab titles
+
+======================================
+Barony Terrain changes:
+--------------------------------------
+Oasis: 4555, 6106,6207,5987,6154,6176,6192, 1431
+Milder Climates: 4558, 4555, 4556
+Plains: 2244, 517, 523 
+Farmland: 4106
+Hills: 519
+Drylands: 1186, 7135, 1424
+
+======================================
+Possiblities:
+--------------------------------------
+* Greco-Afghan culture hybrid in Kandhar (c_rukhaj). Was attested to as late as 100 AD by Isadore of Charax
```

Comment:
I think 100 is a bit early to justify inclusion in 476.

Reply:
Fair enough; I noted it just as an interesting tidbit/potential Easter egg I stumbled across while reading.